cert-data.yml

```diff
@@ -1,6 +1,24 @@
 ---
 # The 4_programmatic certificate renderer uses a special markup language
 #
+# Flags:
+# Font flags need four associated '.ttf' files in Stanford-Online/certificate-templates/template_data/fonts.
```

Comment:
how about: `# Font flags need four associated '.ttf' files in your 'templates/template_data/fonts' directory.` ?